GameEngine/GameEngine/Engine/Culling/Cullable/Object/PointLight.hh

```diff
@@ -17,7 +17,6 @@ namespace AGE
 	{
 		// To remove when light will be in octree
 		uint32_t activePointLightIdx;
-		PointLightData data;
 		glm::mat4 sphereTransform;
 		void computeSphereTransform();
 	};
```

Comment:
I did not clean the code. And I dont want to do that now before we reimplent all we had. But you re right, this is useless :)